### PR TITLE
ft(frontend): implement Logout and fix navigation issues

### LINKED ISSUE #7: Implement logout & Improve app navigation
- You should be able to logout
- Check if current app navigation (moving through different screen) is correct

### PR DESCRIPTION
**Pull request by niyibi250**
### What does this PR do?
This PR ensures that the logout functionality works correctly and addresses other navigation issues within the application.
### Description of Task to be completed?

- [x] Fix the logout functionality to properly terminate the user session.
- [x] Resolve navigation bugs to improve user experience and prevent issues when switching between screens.

### How should this be manually tested?

- Log in to the application using trainee credentials.   
- Navigate to various pages to ensure smooth navigation.
- Navigate to Menu and press to Logout on Menu
- Confirm that the user is redirected to the login page.
- Check that the user cannot access restricted areas after logging out.

Any background context you want to provide?
There were some issues with the logout button not properly ending sessions and notification bell were working like logout which were affecting the user experience. The organization info on the profile were not full scrollable. 

### What is the relevant pivotal tracker/Trello stories?
N/A

### Screenshots (if appropriate)

Screenshot 2024-10-14 115859

Screenshot 2024-10-14 115911

Questions
N/A

### CODE REVIEW

**Review by Christian-Ishimwe — APPROVED** [review]

**Review by MugemaneBertin2001 — APPROVED** [review]

**Review by Mbabazi12 — APPROVED** [review]

**Review by fredshema — APPROVED** [review]

File: components/sidebar.tsx [hunk 1]
```diff
@@ -70,11 +71,25 @@ const Sidebar: React.FC<SidebarProps> = ({ onClose }) => {
     }
   }, [pathname]);
 
+  const handleLogout = async () => {
+    try {
+      await AsyncStorage.removeItem('authToken');
+      router.push('/auth/login');
+    } catch (error) {
+      console.error('Error logging out:', error);
```
Comment (ncutixavier) [inline components/sidebar.tsx]: Remove all logs, or show error alerts

**Review by Mbabazi12 — APPROVED** [review]

File: components/sidebar.tsx (hunk 1 above)
Comment (niyibi250) [reply components/sidebar.tsx]: Done collect that

**Review by MugemaneBertin2001 — APPROVED** [review]

File: components/sidebar.tsx [hunk 2]
```diff
@@ -60,7 +61,7 @@ const Sidebar: React.FC<SidebarProps> = ({ onClose }) => {
   const LowerItems = [
     { name: 'Docs', iconLight: lightDocument, iconDark: darkDocument, path: '/dashboard/trainee' },
     { name: 'Help', iconLight: lightHelp, iconDark: darkHelp, path: '/dashboard/trainee' },
-    { name: 'LogOut', iconLight: lightLogout, iconDark: darkLogout, path: '/dashboard/trainee' },
+    { name: 'LogOut', iconLight: lightLogout, iconDark: darkLogout, path: '/auth/login' },
```
Comment (ncutixavier) [inline components/sidebar.tsx]: Change name to "Sign out"

Comment (niyibi250) [reply components/sidebar.tsx]: Done too

**Review by ncutixavier — APPROVED** [review]

**Pull request merged**